### PR TITLE
[PT] Adding weeks 3 to 6

### PR DESCRIPTION
**Pull request by leonsolon**
I've corrected the rewritten files. Sorry for the trouble! Good to go.

### CODE REVIEW

File: docs/pt/week03/03-1.md [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+
+<!--<center><img src="{{site.baseurl}}/images/week03/03-1/Simon_Thorpe.png" alt="Simon_Thorpe" style="zoom:55%;" /></center>
+-->
+
+<center><img src="{{site.baseurl}}/images/week03/03-1/Simon_Thorpe.png" alt="Simon_Thorpe" style="zoom:55%;" /></center>
+
+<!--<div align="center">Figure 11. Simon Thorpe's model of visual information flow in the brain <div>
+-->
+
+<div align="center">Figura 11. Modelo de Simon Thorpe de fluxo de informações visuais no cérebro<div>
```
Comment (Atcold) [inline docs/pt/week03/03-1.md:395]: Something is wrong here (and also in the English version).
The French version has the correct rendering.

File: docs/pt/week03/lecture03.sbv [hunk 2]
```diff
@@ -0,0 +1,3429 @@
+0:00:04.819,0:00:08.319
+In this case, we have a network which has an input on the left-hand side
+
+0:00:08.959,0:00:14.259
+Usually you have the input on the bottom side or on the left. They are pink in my slides
+
+0:00:14.260,0:00:17.409
+So if you take notes, make them pink. No, just kidding!
```
Comment (Atcold) [inline docs/pt/week03/lecture03.sbv:8]: Uh… This is English?

File: docs/pt/week03/practicum03.sbv [hunk 3]
```diff
@@ -0,0 +1,1751 @@
+0:00:00.020,0:00:07.840
+So convolutional neural networks, I guess today I so foundations me, you know, I post nice things on Twitter
+
+0:00:09.060,0:00:11.060
+Follow me. I'm just kidding
+
+0:00:11.290,0:00:16.649
+Alright. So again anytime you have no idea what's going on. Just stop me ask questions
```
Comment (Atcold) [inline docs/pt/week03/practicum03.sbv:8]: More English…

File: docs/pt/week05/lecture05.sbv [hunk 4]
```diff
@@ -0,0 +1,3572 @@
+0:00:00.000,0:00:04.410
+All right so as you can see today we don't have Yann. Yann is somewhere else
+
+0:00:04.410,0:00:09.120
+having fun. Hi Yann. Okay so today's that we have
+
+0:00:09.120,0:00:13.740
+Aaron DeFazio he's a research scientist at Facebook working mostly on
```
Comment (Atcold) [inline docs/pt/week05/lecture05.sbv:8]: English.

File: docs/pt/week05/practicum05.sbv [hunk 5]
```diff
@@ -0,0 +1,1241 @@
+0:00:00.000,0:00:05.339
+last time we have seen that a matrix can be written basically let me draw here
+
+0:00:05.339,0:00:12.719
+the matrix so we had similar roles right and then we multiplied usually design by
+
+0:00:12.719,0:00:18.210
+one one column all right and so whenever we multiply these guys you can see these
```
Comment (Atcold) [inline docs/pt/week05/practicum05.sbv:8]: English.

File: docs/pt/week06/06-2.md [hunk 6]
```diff
@@ -0,0 +1,586 @@
...
+
+<!--<center>
+ "
+<img src="{{site.baseurl}}/images/week06/06-2/RNN_unrolled.png" /><br>
+Figure 2. Recurrent Networks with unrolled loop
+</center>
+-->
+
+<center>
+ "
```
Comment (Atcold) [inline docs/pt/week06/06-2.md:94]: What is this?

File: docs/pt/week06/06-3.md [hunk 7]
```diff
@@ -0,0 +1,734 @@
...
+<!--RNN is one type of architecture that we can use to deal with sequences of data. What is a sequence? From the CNN lesson, we learned that a signal can be either 1D, 2D or 3D depending on the domain. The domain is defined by what you are mapping from and what you are mapping to. Handling sequential data is basically dealing with 1D data since the domain is the temporal axis. Nevertheless, you can also use RNN to deal with 2D data, where you have two directions.
+-->
+
+RNN é um tipo de arquitetura que podemos usar para lidar com sequências de dados. O que é uma sequência? Com a lição da CNN, aprendemos que um sinal pode ser 1D, 2D ou 3D, dependendo do domínio. O domínio é definido pelo que você está mapeando e para o que está mapeando. Manipular dados sequenciais é basicamente lidar com dados 1D, uma vez que o domínio é o eixo temporal. No entanto, você também pode usar RNN para lidar com dados 2D, onde você tem duas direções.
+
+<!--
+### Vanilla *vs.* Recurrent NN
+-->
+
+### Rede Neural "Comum" * vs. * Redes Neurais Recorrentes
```
Comment (Atcold) [inline docs/pt/week06/06-3.md:28]:
```suggestion
### Rede Neural "Comum" *vs.* Redes Neurais Recorrentes
```

File: docs/pt/week06/lecture06.sbv [hunk 8]
```diff
@@ -0,0 +1,3338 @@
+0:00:04.960,0:00:08.970
+So I want to do two things, talk about
+
+0:00:11.019,0:00:14.909
+Talk a little bit about like some ways to use Convolutional Nets in various ways
+
+0:00:16.119,0:00:18.539
+Which I haven't gone through last time
```
Comment (Atcold) [inline docs/pt/week06/lecture06.sbv:8]: English.

File: docs/pt/week06/practicum06.sbv [hunk 9]
```diff
@@ -0,0 +1,1742 @@
+0:00:00.030,0:00:03.959
+so today we are gonna be covering quite a lot of materials so I will try not to
+
+0:00:03.959,0:00:08.309
+run but then yesterday young scooped me completely so young talked about exactly
+
+0:00:08.309,0:00:15.269
+the same things I wanted to talk today so I'm gonna go a bit faster please slow
```
Comment (Atcold) [inline docs/pt/week06/practicum06.sbv:8]: English.

**Review by Atcold — CHANGES_REQUESTED** [review]
Wow, I just realised you sent a cumulative PR with other authors.
You need to add them to the commit using the `coauthor` field.
Also, see review feedback.

**Comment by Atcold** [comment]
Any update on this?

**Comment by dsantiago** [comment]
> Any update on this?



> Any update on this?

Maybe i can help here if others are too busy...

**Comment by Atcold** [comment]
I believe you don't have writing access to @leonsolon's fork.

**Comment by dsantiago** [comment]
If needed i can create one my own...

**Comment by leonsolon** [comment]
Sorry for the delay! Let me know if there's anything left unfixed.

**Comment by Atcold** [comment]
Awesome, thanks!

**Pull request merged**